server.lua

```diff
@@ -437,7 +449,13 @@ RegisterNetEvent('qb-vehicleshop:server:checkFinance', function()
         local vehicles = MySQL.query.await(query, {player.PlayerData.citizenid})
         for _, v in pairs(vehicles) do
             local plate = v.plate
+            local financetime = MySQL.query.await('SELECT financetime FROM player_vehicles WHERE plate = ?', {plate})
+            local balance = MySQL.query.await('SELECT balance FROM player_vehicles WHERE plate = ?', {plate})
```

Comment:
3 seperate queries inside a loop is not ideal
This can be easily done as one query